dask/dataframe/io/tests/test_parquet.py

```diff
@@ -597,7 +597,7 @@ def test_roundtrip_nullable_dtypes(tmp_path, write_engine, read_engine):
 @pytest.mark.parametrize(
     "dtype_backend",
     [
-        "pandas",
+        "numpy_nullable",
```

Comment:
Given this test is for the old (now deprecated) `dataframe.dtype_backend` config option, I think we want to keep this `"pandas"` not `"numpy_nullable"` (the config option will never be set to `"numpy_nullable"`)